apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/event/api/dto/AllowedToCreateDto.java

```diff
@@ -0,0 +1,23 @@
+package edu.pw.apsienrollment.event.api.dto;
+
+import edu.pw.apsienrollment.user.db.UserRole;
+import lombok.Builder;
+import lombok.Value;
+import lombok.NonNull;
+
+import edu.pw.apsienrollment.event.db.EventType;
+
+import java.util.List;
+import java.util.Map;
+
+@Value
+@Builder
+public class AllowedToCreateDto {
+    Map<EventType, List<UserRole>> allowedToCreate;
+
+    public static AllowedToCreateDto of(@NonNull Map<EventType, List<UserRole>> allowedToCreate) {
```

Comment:
I think this dto could contain <EventType, Collection> - most java data structures implement Collection interface, so it would simplify the service method. We could get rid of `stream().collect(Collectors.toList())`. And it will not make any difference to client.